bin/webpack-entries.js

```diff
@@ -2,19 +2,139 @@
  * External dependencies
  */
 const { omit } = require( 'lodash' );
+const glob = require( 'glob' );
 
-const stable = {
-	core: {
-		wcBlocksRegistry: './assets/js/blocks-registry/index.js',
-		wcSettings: './assets/js/settings/shared/index.js',
-		wcBlocksData: './assets/js/data/index.js',
-		wcBlocksMiddleware: './assets/js/middleware/index.js',
-		wcSharedContext: './assets/js/shared/context/index.js',
+const blocks = {
+	'handpicked-products': {
+		dir: 'handpicked-products',
 	},
-	main: {
-		// Shared blocks code
-		blocks: './assets/js/index.js',
+	'product-best-sellers': {
+		dir: 'product-best-sellers',
+	},
+	'product-category': {
+		dir: 'product-category',
+	},
+	'product-categories': {
+		dir: 'product-categories',
```

Comment:
Is there some way we could automate this map?

Reply:
I did a small refactor in b4d96edcb5e193b790fa7f4620d5328beb483303 so we are less explicit about directories and instead they are inferred from the block name.